sdk/core/abort-controller/test/aborter.spec.ts

```diff
@@ -1,7 +1,9 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import * as assert from "assert";
+import { assert, use as chaiUse } from "chai";
+import chaiPromises from "chai-as-promised";
+chaiUse(chaiPromises);
```

Comment:
why do we need this? Unless a test uses `eventually`, there is no point to this, and really I think using `await` obviates `eventually` completely.

Reply:
we use it for testing rejected promises in TA and I thought this could be useful but I see your point that it is not even used here. I removed it.